tiling.js

```diff
@@ -1485,8 +1486,14 @@ class Spaces extends Map {
         let from = workspaceManager.get_workspace_by_index(fromIndex);
         let toSpace = this.spaceOf(to);
 
-        if (!inPreview)
-            this._initWorkspaceStack();
+        if (!inPreview) {
+            if (inPreviewMode.stack) {
+                this._initWorkspaceStack();
+            }
+            else if (inPreviewMode.sequence) {
+                this._initWorkspaceSequence();
```

Comment:
I think it makes sense to default to the sequential animation.

Reply:
done